server/routes.go

```diff
@@ -80,8 +80,9 @@ type RESTServer struct {
 	Fixity        FixityDB
 	DisableFixity bool
 
-	server httpdown.Server // used to close our listening socket
-	txgate *util.Gate      // limits number of concurrent transactions
+	server   httpdown.Server // used to close our listening socket
+	txgate   *util.Gate      // limits number of concurrent transactions
+	TapeUsed bool            // Is Bendo reading/writing from tape?
```

Comment:
Make this an unexported structure member.

I also like keeping the name similar to the variable in `items`. So rename to `useTape`?

This variable technically needs a mutex, since it can be changed once the server is running. I am only pointing it out for completeness, and I am not asking you to guard it.